### PR TITLE
Update hacking.md to get pprof data for OpenShift node.

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
This updates the documentation for profiling OpenShift 3.10. 

We cannot get the pprof data for node any more using the existing method. See https://bugzilla.redhat.com/show_bug.cgi?id=1572919

/cc @sjenning @wabouhamad

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -776,14 +776,27 @@ not correctly sampled
...
   * `web` - start the pprof webserver in process at http://127.0.0.1:6060/debug/pprof
 (you can open this in a browser). This supports `OPENSHIFT_PROFILE_HOST=`
-and `OPENSHIFT_PROFILE_PORT=` to change default ip `127.0.0.1` and default port `6060`.
+and `OPENSHIFT_PROFILE_PORT=` to change default ip `127.0.0.1` and default port `6060`. 
+This works for master components only.
+  * In case of openshift-node, we need to get data from port on which kubelet is running 10250(default port). We can
+   get the pprof data using following command:
+   
+
+   	`curl --cert /tmp/admin.crt --key /tmp/admin.key https://<node_name>:10250/debug/pprof/profile  -k --output /tmp/sample.pprof -L`
```
Comment (smarterclayton) [inline HACKING.md]: From any admin kubeconfig credentials, you can do:

    $ oc get --raw /debug/pprof/profile --server https://node_name:10250 > /tmp/sample.pprof

which is easier.

Comment (ravisantoshgudimetla) [reply HACKING.md]: That is correct and I can update the documentation with both the ways.

File: HACKING.md [hunk 2]
```diff
@@ -776,14 +776,27 @@ not correctly sampled
...
+This works for master components only.
+  * In case of openshift-node, we need to get data from port on which kubelet is running 10250(default port). We can
+   get the pprof data using following command:
+   
+ 	`oc get --raw /debug/pprof/profile --server https://node_name:10250 > /tmp/sample.pprof`	
+	
+	  `profile` in the above URL could be replaced with `heap`, `block` or `mutex`.
+    * We can open the file using:
+      
+  	$ go tool pprof /tmp/sample.pprof
```
Comment (sjenning) [inline HACKING.md]: put this in a code block like the others w/o prompt $

File: HACKING.md [hunk 3]
```diff
@@ -776,14 +776,27 @@ not correctly sampled
   * `block` -  will start a block wait time analysis and write `./block.pprof`
   * `web` - start the pprof webserver in process at http://127.0.0.1:6060/debug/pprof
 (you can open this in a browser). This supports `OPENSHIFT_PROFILE_HOST=`
-and `OPENSHIFT_PROFILE_PORT=` to change default ip `127.0.0.1` and default port `6060`.
+and `OPENSHIFT_PROFILE_PORT=` to change default ip `127.0.0.1` and default port `6060`. 
+This works for master components only.
+  * In case of openshift-node, we need to get data from port on which kubelet is running 10250(default port). We can
```
Comment (sjenning) [inline HACKING.md]: nit: space between 10250 and (default port)

**Comment by sjenning** [comment]
couple of comment, then lgtm

**Pull request closed without merging**